lib/composer.js

```diff
@@ -333,7 +336,7 @@ export default class Composer extends Disposable {
     }
 
     if (this.shouldUseDiCy()) {
-      return this.runDiCy(['load', 'build', 'log', 'save'], { ignoreCache: shouldRebuild },
+      return this.runDiCy(['load', 'build', 'log', 'save'], { ignoreCache: !!shouldRebuild },
```

Comment:
This seems unnecessary given the argument has a default value of `false`.

Reply:
Good point.